templates/LABELS.md

```diff
@@ -0,0 +1,59 @@
+# Available labels
+
+## Operating systems
+
+The operating system labels must match the [libosinfo
+identifiers](https://gitlab.com/libosinfo/osinfo-db/tree/master/data/os) from the relevant xml files (look for short-id).
+
+### Ubuntu
+
+- os.template.cnv.io/ubuntu18.04
+- os.template.cnv.io/ubuntu17.10
+- os.template.cnv.io/ubuntu17.04
+- os.template.cnv.io/ubuntu16.10
+- os.template.cnv.io/ubuntu16.04
+
+### Fedora
+
+- os.template.cnv.io/fedora29
+- os.template.cnv.io/fedora28
+- os.template.cnv.io/fedora27
+- os.template.cnv.io/fedora26
+- os.template.cnv.io/fedora25
+- os.template.cnv.io/fedora24
+- os.template.cnv.io/fedora23
+
+### CentOS
+
+- os.template.cnv.io/centos7.0
+- os.template.cnv.io/centos6.9
+- os.template.cnv.io/centos6.8
+- os.template.cnv.io/centos6.7
+
+### Microsoft Windows
+
+- os.template.cnv.io/win2k8r2
+- os.template.cnv.io/win2k8
+- os.template.cnv.io/win10
+- os.template.cnv.io/win8.1
+- os.template.cnv.io/win8
+- os.template.cnv.io/win7
+- os.template.cnv.io/winvista
+- os.template.cnv.io/winxp
+
+## Workload profiles
```

Comment:
- we currently have no distinction between server and desktop in CNV, but I can obviously do that easily
- sap-hana is an example and we do not have to use it just yet
- high-performance is CPU intensive load vs. io-intensive is disk performance, those two do not have to be the same, right?

Reply:
I wouldn't give anyone any ideas yet. We should build workloads as we go. Too many would be confusing.

if high performance == cpu intensive and we're thinking of adding i/o instensive, doesn't it make sense to call "cpu intensive" or some other word, rather than "high performance" which is too generic and makes sense when you only have one

Reply:
Sure.